readme.md

```diff
@@ -7,7 +7,7 @@ The NVDA screen reader is a 32 bit application and requires the 32 bit Java Acce
 This will happily talk to 64 bit Java applications.
 However, since Java 10, official 32 bit builds are no longer provided.
 
-This file was extracted from a Zulu Community build of OpenJDK as follows:
+These files were extracted from a Zulu Community build of OpenJDK as follows:
 
 1. Downloaded Java 17.0.9+8Zulu (17.46.19) zip file from: https://www.azul.com/downloads/?os=windows&architecture=x86-32-bit&package=jre#zulu
-2. Copied `zulu17.46.19-ca-jre17.0.9-win_i686\bin\windowsaccessbridge-32.dll` from this archive into this repository.
+2. Copied `zulu17.46.19-ca-jre17.0.9-win_i686\bin\windowsaccessbridge-32.dll` and `zulu17.46.19-ca-jre17.0.9-win_i686\bin\windowsaccessbridge.dll` from this archive into this repository.
```

Comment:
It looks like the download on the web page has been updated to `zulu17.50.19-ca-jre17.0.11-win_i686` can you confirm if you used this new version or the old version?
If you used the new version, please update these docs as well as the other DLL file, so that they both match the new version.